server/views/pages/goal/update/review.njk

```diff
@@ -77,7 +77,7 @@
         <tbody class="govuk-table__body">
           {% for step in data.steps %}
           <tr class="govuk-table__row">
-            <th scope="row" class="govuk-table__header">Step {{ step.stepNumber }}</th>
+            <th scope="row" class="govuk-table__header">Step {{loop.index}}</th>
```

Comment:
Using `loop.index` here as the `stepNumber` has not been updated in the database at this point, as the user should review the answers before saving the data.

Reply:
If a goal has 3 steps and I remove step 2,
The `step.stepNumber` in the database for step 3 is still `3`, until I save the form on the review screen
Using the `step.stepNumber` as a table header on the review screen here, will display 
Step 1 - Value
Step 3 - Value

As the `step.stepNumber` hasn't been updated yet

Reply:
Ah, I see. In that case we need to re-jig / re-sequence the numbers as part of removing a step in the controller method - so that they array of steps are always sequential 👍